### ISSUE TITLE
Unknown encoder 'libfaac'

### ISSUE THREAD

**Issue by Maliron** [maddox/wallop#70]
So... I compiled ffmpeg from source and tried to add libfaac, but it seems that libfaac support has been completely removed from ffmpeg.

http://git.videolan.org/?p=ffmpeg.git;a=commit;h=dc0f711459e0c682bf9f94ba38d26736e90cff45

It seems though that the HDHomeRun Prime is outputting FAAC audio, and the wallop.rb function is calling to just copy the audio streams, so ffmpeg is failing everything when using more modern versions.. any ideas how we can fix this? Maybe I'm not understanding the problem 100% either. Looking at the log looks like the wallop is trying to force faac encoding.. Here is a slice of my ffmpeg log:

Input #0, mpegts, from 'http://192.168.11.138:5004/auto/v1021':
  Duration: N/A, start: 20007.707911, bitrate: N/A
  Program 1020 
    Stream #0:0[0x1661], 64, 1/90000: Video: mpeg2video (Main), 1 reference frame ([2][0][0][0] / 0x0002), yuv420p(tv, top first, left), 1920x1080 [SAR 1:1 DAR 16:9], 0/1, Closed Captions, 29.97 fps, 29.97 tbr, 90k tbn, 59.94 tbc
    Stream #0:1[0x1662](eng), 65, 1/90000: Audio: ac3 ([129][0][0][0] / 0x0081), 48000 Hz, stereo, fltp, 384 kb/s
    Stream #0:2[0x1663](spa), 65, 1/90000: Audio: ac3 ([129][0][0][0] / 0x0081), 48000 Hz, stereo, fltp, 384 kb/s
    Stream #0:3[0x1664], 0, 1/90000: Data: scte_35, 0/1
Successfully opened the file.
Parsing a group of options: output url /home/rreese/wallop/tmp/1021.m3u8.
Applying option ac (set number of audio channels) with argument 2.
Applying option acodec (force audio codec ('copy' to copy stream)) with argument libfaac.
Applying option b:v (video bitrate (please use -b:v)) with argument 3000k.
Applying option vcodec (force video codec ('copy' to copy stream)) with argument libx264.
Applying option s (set frame size (WxH or abbreviation)) with argument 1280x720.
Applying option r (set frame rate (Hz value, fraction or abbreviation)) with argument 29.97.
Applying option map (set input stream mapping) with argument 0:0.
Applying option map (set input stream mapping) with argument 0:1.
Successfully parsed a group of options.
Opening an output file: /home/rreese/wallop/tmp/1021.m3u8.
**Unknown encoder 'libfaac'**

**Comment by tmm1** [comment]
You can replace libfaac with aac

**Comment by Maliron** [comment]
Yeah, I'm just now seeing that in the config file.. lol. Now I feel dumb... heh Thanks for the reply.
